### DISCUSSION TITLE
WiFi power management off by default?

### DISCUSSION THREAD

**Discussion by Pjotr123** [linuxmint/discussions#497] [Ideas]
A recurring headache is the overactive WiFi power management, which creates WiFi instability reports all over the Linux Mint forum (and has done so for years). While the fix is easy:
sudo sed -i 's/3/2/' /etc/NetworkManager/conf.d/default-wifi-powersave-on.conf

Please apply that fix in the upcoming Mint 22.

**Comment by LWFlouisa** [comment]
I've been having the issue of it automatically turning off. Along with things like Operating system freezes just before Linux Updates in the graphical GUI.

"Hardware Issues" simply doesn't explain all the freezing issues I have on certain websites like Pinterest and Youtube.

Specifically it freezes for so long I have to restart my computer.
